R/find_rev.R

```diff
@@ -1138,8 +1146,22 @@ nix_rprofile <- function() {
 #' )
 #' 
 #' # There no limit in the complexity of function call stacks that `with_nix()`
-#' # can possibly handle
+#' # can possibly handle; however, `expr` should not evaluate and 
+#' # needs to be a function for `program = "R"`. If you want to pass the
+#' # a function with arguments, you can do like this
+#' get_sample <- function(seed, n) {
+#'   set.seed(seed)
+#'   sample(seq(1, 10), n)
+#'   set.seed(NULL)
```

Comment:
This function returns NULL, the output of sample() should be saved in a variable and this variable returned instead